server/json-rpc/src/lib.rs

```diff
@@ -67,7 +67,7 @@ where
                         json_rpc_error(JSON_RPC_CREATE_ERROR, e.to_string())
                     }
                 })
-                .map(move |_| Value::Null),
+                .map(move |id| Value::String(id)),
```

Comment:
How about returning `{"id": ...}` here so we can add more properties without breaking compatibility? In case external graph-node users decide to use it